troposphere/emr.py

```diff
@@ -122,15 +122,76 @@ class EbsConfiguration(AWSProperty):
     }
 
 
+class Constraints(AWSProperty):
+    props = {
+        'MinCapacity': (integer, False),
```

Comment:
These are both required per the docs: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-emr-cluster-jobflowinstancesconfig-instancegroupconfig-autoscalingpolicy-constraints-scalingconstraints.html